<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>balancing_programmClass</class>
 <widget class="QMainWindow" name="balancing_programmClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>水准网平差程序</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="method">
    <property name="title">
     <string>平差方法</string>
    </property>
    <addaction name="actioncod_adjust"/>
    <addaction name="separator"/>
    <addaction name="actionird_adjust"/>
   </widget>
   <widget class="QMenu" name="display">
    <property name="title">
     <string>显示</string>
    </property>
    <addaction name="actionori_data"/>
    <addaction name="separator"/>
    <addaction name="actionleve_network"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionhelp"/>
   </widget>
   <addaction name="file"/>
   <addaction name="method"/>
   <addaction name="display"/>
   <addaction name="help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="icon">
    <iconset resource="balancing_programm.qrc">
     <normaloff>:/Images/打开.png</normaloff>:/Images/打开.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="balancing_programm.qrc">
     <normaloff>:/Images/保存.png</normaloff>:/Images/保存.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="actioncod_adjust">
   <property name="text">
    <string>条件平差</string>
   </property>
  </action>
  <action name="actionird_adjust">
   <property name="text">
    <string>间接平差</string>
   </property>
  </action>
  <action name="actionori_data">
   <property name="text">
    <string>原始数据</string>
   </property>
  </action>
  <action name="actionleve_network">
   <property name="text">
    <string>水准网</string>
   </property>
  </action>
  <action name="actionresult">
   <property name="text">
    <string>平差结果</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="balancing_programm.qrc"/>
 </resources>
 <connections/>
</ui>
